test/fixtures/expect.go

```diff
@@ -46,7 +46,7 @@ func (t *Expect) SinkContains(sinkName string, targetStr string, opts ...SinkChe
 	ctx := context.Background()
 	contains := RedisContains(ctx, t.pipeline.Name, sinkName, targetStr, opts...)
 	if !contains {
-		t.t.Fatalf("Expected redis contains target string %s written by pipeline %s, sink %s.", targetStr, t.pipeline.Name, sinkName)
+		t.t.Fatalf("Expected redis doesn't contain target string %s written by pipeline %s, sink %s.", targetStr, t.pipeline.Name, sinkName)
```

Comment:
shouldn't this still be contains ?

Reply:
this is when the string is not found in sink pod

Reply:
I think in `expect.go`, we use the following pattern to print fatals. 
```
if Not A {
		t.t.Fatalf("Expected A")
	}
```

Reply:
reverted